### PR TITLE
Adds detection for CoolBrowser

### PR DESCRIPTION
**Pull request by liviuconcioiu**

### CODE REVIEW

**Comment by liviuconcioiu** [comment]
There is another user agent with `CoolBrowser`, but I don't know if is the same browser or not.
`Mozilla/5.0 (Linux; Android 4.4.2; i-mobile_i-STYLE_219 Build/KOT49H) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 CoolBrowser/30.0.0.0 Mobile Safari/537.36`

**Comment by liviuconcioiu** [comment]
@sanchezzzhak I think that user agent is from a Android trojan.

`com.sailer.coolbrowser`
http://dl.cm.ksmobile.com/static/res/24/15/1606271wl.db

File: Tests/Parser/Client/fixtures/browser.yml [hunk 1]
```diff
@@ -2696,35 +2696,11 @@
...
     engine_version: "537.36"
 -
-  user_agent: Mozilla/5.0 (Linux x86_64) EkiohFlow/5.11.1.33219 Flow/5.11.1 (like Gecko Firefox/62.0 rv:62.0)
+  user_agent: Mozilla/5.0 (Linux; Android 10; JNY-LX1 Build/HUAWEIJNY-L21; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/78.0.3904.108 Mobile Safari/537.36 AgentWeb/4.1.3 UCBrowser/11.6.4.950
   client:
     type: browser
-    name: Flow
-    version: 5.11.1
-    engine: EkiohFlow
-    engine_version: 5.11.1.33219
```
Comment (sanchezzzhak) [inline Tests/Parser/Client/fixtures/browser.yml:2705]: check if this test was deleted accidentally

**Comment by sanchezzzhak** [comment]
I test in emulator device, useragent  result
```
Mozilla/5.0 (Linux; Android 11; sdk_gphone_x86_64 Build/RSR1.201211.001; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/83.0.4103.106 Mobile Safari/537.36 AgentWeb/4.1.3 UCBrowser/11.6.4.950
```

**Review by sanchezzzhak — APPROVED** [review]

**Pull request merged**